### PR TITLE
Update OpenIdConnectRequestType

### PR DESCRIPTION
**Pull request by ChenYan98**
#373

### CODE REVIEW

File: src/Microsoft.IdentityModel.Protocols.OpenIdConnect/OpenIdConnectRequestType.cs [hunk 1]
```diff
@@ -34,9 +34,9 @@ namespace Microsoft.IdentityModel.Protocols.OpenIdConnect
     public enum OpenIdConnectRequestType
     {
 #pragma warning disable 1591
```
Comment (polita) [inline src/Microsoft.IdentityModel.Protocols.OpenIdConnect/OpenIdConnectRequestType.cs]: Can we pull out this disabling of the compiler warning and correctly xml-doc this member?

File: src/Microsoft.IdentityModel.Protocols.OpenIdConnect/OpenIdConnectResponseMode.cs [hunk 2]
```diff
@@ -30,7 +30,7 @@ namespace Microsoft.IdentityModel.Protocols.OpenIdConnect
     /// <summary>
     /// Response modes for OpenIdConnect.
     /// </summary>
-    public static class OpenIdConnectResponseModes
+    public static class OpenIdConnectResponseMode
     {
 #pragma warning disable 1591
```
Comment (polita) [inline src/Microsoft.IdentityModel.Protocols.OpenIdConnect/OpenIdConnectResponseMode.cs]: Same as above.

Comment (brentschmaltz) [inline src/Microsoft.IdentityModel.Protocols.OpenIdConnect/OpenIdConnectResponseMode.cs]: Will need to add comments as it is public.

Comment (ChenYan98) [inline src/Microsoft.IdentityModel.Protocols.OpenIdConnect/OpenIdConnectResponseMode.cs]: Added

File: src/Microsoft.IdentityModel.Protocols.OpenIdConnect/OpenIdConnectRequestType.cs (hunk 1 above)
Comment (ChenYan98) [inline src/Microsoft.IdentityModel.Protocols.OpenIdConnect/OpenIdConnectRequestType.cs]: Added comments as it is public.

Comment (polita) [inline src/Microsoft.IdentityModel.Protocols.OpenIdConnect/OpenIdConnectRequestType.cs]: Can we remove the pragmas then?

**Comment by brentschmaltz** [comment]
:shipit: You will need to rebase before merging.

**Comment by polita** [comment]
+1

**Pull request merged**